### DISCUSSION TITLE
Is the MySQL driver provided by default?

### DISCUSSION THREAD

**Discussion by LiuTianyou** [apache/hertzbeat#2080] [General]
Hello, community

I encountered an issue while deploying version 1.6.0 of Hertzbeat using Docker. Without any special configuration, I receive the following error when setting up MySQL monitoring:

```
Query Error: No suitable driver found for jdbc:mysql://192.168.75.239:3306/?useUnicode=true&characterEncoding=utf-8&useSSL=false Code: 0
```
<img width="1429" alt="image" src="https://github.com/apache/hertzbeat/assets/30208283/5b518236-565a-4510-acd2-6bcac28bb09a">

Mounting the MySQL driver to the `ext-lib` directory of the container resolves this issue. This can be confusing for users unfamiliar with Hertzbeat, especially since the MySQL monitoring documentation does not mention the requirement to add the driver to this directory.

Here are two potential solutions:

1. Provide the MySQL driver by default.
2. Clearly document in the MySQL monitoring guide that adding the MySQL driver to the ext-lib directory is necessary.

What are your thoughts and suggestions on this issue?

**Comment by tomsun28** [comment]
hi, we can not provide the MYSQL jdbc driver by default in release package due the ASF 3RD PARTY LICENSE POLICY https://www.apache.org/legal/resolved.html. The MYQL jdbc driver license is not compatible with Apache2 License, same with Oracle jdbc driver. 
Suggest we can add clearly document in the MySQL monitoring guide to note the necessary for user to add ext-lib by themself.
